src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1990,7 +1990,13 @@ default MessageCreateAction replyFiles(@Nonnull Collection<? extends FileUpload>
         /**
          * Represents a mention for all users in a server, literal {@code @everyone}.
          */
-        EVERYONE("@everyone", "everyone");
+        EVERYONE("@everyone", "everyone"),
+        /**
+         * Represents a mention for a slash command.
+         * <br>The first group is the command name, the second group is the subcommand group (nullable),
+         * the third group is the subcommand (nullable), and the fourth group is the command ID.
+         */
+        SLASH_COMMAND("</([\\w-]+)(?> ([\\w-]+))??(?> ([\\w-]+))?:(\\d+)>", null);
```

Comment:
Don't know what Minn and DV8 think about this, but I feel that group names are a lot more readable
```suggestion
        SLASH_COMMAND("</(?<commandName>[\\w-]+)(?<subcommandGroup>> ([\\w-]+))??(?<subcommandName>> ([\\w-]+))?:(?<commandId>\\d+)>", null);
```

Reply:
Other regexes there don't do that

Reply:
As a note, the other regexes are preeeetty basic which is why they don't do this. there's no big reason to not do it 🤷